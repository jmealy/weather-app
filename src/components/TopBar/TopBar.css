.TopBar {
  height: 50px;
  background-color: #f63;
  display: flex;
  flex-direction: row;
}

.SearchArea {
  line-height: 50px;
  margin-left: auto;
  margin-right: 20px;
}

.appTitle {
  margin: 0;
  padding: 0 20px;
  line-height: 50px;
  text-align: left;
  color: white;
}