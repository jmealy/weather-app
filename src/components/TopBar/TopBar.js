import React from 'react';
import LocationSearch from '../LocationSearch/LocationSearch';
import './TopBar.css';

const Header = () => {
  return (
    <div className="TopBar">
      <div className="appTitle">Weather App dawg</div>
      <div className="SearchArea">
        <LocationSearch></LocationSearch>
      </div>

    </div>
  );
}

export default Header;
