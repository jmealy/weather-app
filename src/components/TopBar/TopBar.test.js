import React from 'react';
import renderer from 'react-test-renderer';
import TopBar from './TopBar';

test('renders learn react link', () => {
  const tree = renderer.create(<TopBar onLocationSubmit={jest.fn()} />);
  expect(tree).toMatchSnapshot();
});
