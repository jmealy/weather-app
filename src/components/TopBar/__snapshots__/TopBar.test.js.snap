// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="topBar"
>
  <div
    className="appTitle"
  >
    Weather App
  </div>
  <div
    className="topBarSearch"
  >
    <form
      className="formContainer"
      onSubmit={[Function]}
    >
      <input
        className="textInput"
        placeholder="London"
        type="text"
      />
      <input
        className="submitButton"
        type="submit"
        value="Get Weather"
      />
    </form>
  </div>
</div>
`;
